include/Color.hpp

```diff
@@ -243,6 +240,37 @@ class Color : public ::Color {
     static Color Blank() { return BLANK; }
     static Color Magenta() { return MAGENTA; }
     static Color RayWhite() { return RAYWHITE; }
+    */
+
+    /**
+     * As an alternative to a function calling convention
+     */
```

Comment:
This will become the doxygen docs for LightGray. It might be best to just remove the block as it's pretty clear what it does :sunglasses: 

``` suggestion
```